checkmarble/marble-frontend
# feat(auth): add login with email

Add a login with email/password to the app

To test :
1. ensure you have created a user
  a. on Firebase, using the console (ex for stagging go [here](https://console.firebase.google.com/u/0/project/tokyo-country-381508/authentication/users)
  b. on the BO
2. go to `/login-with-email`

> NB: error messages are less clear when leaving emulator due to [this recomandation](https://firebase.google.com/docs/auth/web/password-auth#enumeration-protection)

![output](https://github.com/checkmarble/marble-frontend/assets/40292402/8b4ccda5-bccb-4fb2-90df-8c46b3c0360a)
